docs/src/components/sidebar/treeNode.js

```diff
@@ -26,7 +26,7 @@ const TreeNode = ({ className = '', setCollapsed, collapsed, url, title, items,
   return (
     <li className={calculatedClassName}>
       {/* TODO: Remove the below url check, its just to hide the item for now */}
```

Comment:
We could remove this comment now too?

Reply:
Ahh I did this, put it back cos there was an error then forgot to remove it, what a wally. Will remove now